frontend/src/layouts/default.vue

```diff
@@ -11,16 +11,26 @@
 
 <script setup lang="ts">
 import { ref, watch, provide } from 'vue';
+import { useRoute } from 'vue-router';
 import { useDisplay } from 'vuetify';
 import { userLibrariesStore } from '@/store';
 
 const display = useDisplay();
+const route = useRoute();
 const userLibraries = userLibrariesStore();
 const navDrawer = ref(!display.mobile.value);
 
-if (!userLibraries.isReady) {
-  await userLibraries.refresh();
-}
```

Comment:
This check must be in place because we also [load the content on login ](https://github.com/jellyfin/jellyfin-vue/blob/vite/frontend/src/components/Forms/LoginForm.vue#L108)

(Forgot to tell when we discussed this in Matrix, sorry!)

Reply:
If I keep the check, the libraries won't be refreshed when your account is remembered and you first load the app.

Reply:
No, because we're using sessionStorage, so the data at userLibraries will be at it's defaults (entering this statement into effect when an user is logged in and you access the app) :)